test/api/test-set.c

```diff
@@ -1067,6 +1067,40 @@ test_set_inverted_operations (void)
   g_assert (all_succeeded);
 }
 
+static void
+test_create_from_sorted (void)
+{
+  hb_codepoint_t array[7] = {1, 2, 3, 1000, 2000, 2001, 2002};
+  hb_set_t *set = hb_set_create_from_sorted (array, 7);
+  g_assert_cmpint (hb_set_get_population (set), ==, 7);
+  g_assert (hb_set_has (set, 1));
+  g_assert (hb_set_has (set, 2));
+  g_assert (hb_set_has (set, 3));
+  g_assert (hb_set_has (set, 1000));
+  g_assert (hb_set_has (set, 2000));
+  g_assert (hb_set_has (set, 2001));
+  g_assert (hb_set_has (set, 2002));
+}
+
+static void
+test_export_to_array (void)
+{
```

Comment:
Once you add a size limit to the export call, add a test with too small of a buffer.

Reply:
Done